.github/workflows/stale.yml

```diff
@@ -0,0 +1,43 @@
+name: close stale issues and pr
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          days-before-issue-stale: 28
+          days-before-issue-close: 7
+          any-of-labels:
+            [
+              "Waiting on author",
+              "Cannot reproduce 🤷",
+              "python past end of life",
+              "Won't fix/not planned",
+              "Work in progress",
```

Comment:
Just because a linked PR is stale doesn't mean the issue should be closed. See issues like https://github.com/pylint-dev/pylint/issues/2296.

```suggestion
```

Reply:
I'd say `"Waiting on author"` is also debatable. I'd prefer closing this manually as we can then also add a nicer message thanking the contributor for their time and pointing them to what they can do if they want to re-open the PR.

Reply:
We're still free to do that if we beat the bot to it!